src/core/ext/filters/client_channel/resolver/binder/README.md

```diff
@@ -0,0 +1,4 @@
+Support for resolving the scheme used by binder transport implementation.
+
+The URI's authority is used as the identifiers of endpoint binder objects.
```

Comment:
I don't think it makes sense to put the identifier of the thing you want to connect to in the *authority* of the URI.  The way that gRPC URIs work, you generally want to put that identifier into the *path* of the URI.  The authority is actually intended to indicate the authority you use to look up the name, not the name itself.  For example, for "dns:" URIs, the authority indicates what DNS server to use, and the path indicates the name to resolve.  Your scheme doesn't need to support an authority at all (we have others that don't), but I think the name that you want to connect to should be in the path.

Reply:
Thanks! the explanation makes sense (I thought I does not matter)

Done